mergify_engine/context.py

```diff
@@ -40,6 +42,18 @@
 
 
 SUMMARY_SHA_EXPIRATION = 60 * 60 * 24 * 31  # ~ 1 Month
+DEPENDS_ON_RE = re.compile(
+    r"^Depends-On: (https://github.com/(?P<owner>[^/]+)/(?P<repo>[^/]+)/pull/|#)(?P<number>\d+)"
+)
+
+
+# NOTE(sileht): In case a pull request is waiting on other, we got notified
+# of dependency states change only when dependency get merged (because
+# we received a push event that will refresh our PR).
+# That why we only handles these states (and not open/close/...)
```

Comment:
```suggestion
# NOTE(sileht): In case a pull request is waiting on others, Mergify is notified
# of dependency states change only when dependency get merged (because
# it receives a `synchronize` push event that refreshes the PR).
# That's why we only handles these states (and not open/close/...)
```